### PR TITLE
Home Buttons Have More Contrast When Keyboard Focused (#803)

### PR DESCRIPTION
**Pull request by vjcao**
This PR addresses issue #803 by making the making the home buttons have more contrast when using the keyboard to navigate.

issue_803

### CODE REVIEW

File: assets/src/containers/DashboardAppBar.js [hunk 1]
```diff
@@ -62,20 +62,11 @@ function DashboardAppBar (props) {
...
-              <Link
-                to={{ pathname: `/courses/${courseId}` }}
-                className={classes.homeButton}>
-                {courseName}
-              </Link>
-            </Typography>
+          <Button href='/courses/' className={classes.homeButton} style={{ fontSize: '1.25rem' }}>
+            My Learning Analytics:
+          </Button>
+          <Button href={`/courses/${courseId}`} className={classes.homeButton} style={{ fontSize: '1.25rem' }}>
```
Comment (ssciolla) [inline assets/src/containers/DashboardAppBar.js]: Can you explain why the `fontSize` was added? Was it to keep the appearance consistent with before? If it is necessary, I think I'd like to see it added to the `homeButton` class that starts on line 28.

Comment (vjcao) [reply assets/src/containers/DashboardAppBar.js]: Fixed

**Comment by jonespm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/ITS_Teaching_And_Learning/student-dashboard-django/pullRequest?prid=4552421&bid=15313922)

**Comment by vjcao** [comment]
@ssciolla This branch currently does not run properly on localhost so I will try to figure out what is wrong before merging.

**Comment by vjcao** [comment]
Merging. The issue was the show_grade_type column missing in SQL test file.

**Pull request merged**